Documentation Changes for v3

gsteel opened the pull request.

gsteel committed:
Remove `cookbook/setting-module-specific-layouts.md`

This page is MVC specific and v3 of `view` is unlikely to gain compatibility with MVC

Signed-off-by: George Steel <george@net-glue.co.uk>

gsteel committed:
Re-Write the Quick-Start

Signed-off-by: George Steel <george@net-glue.co.uk>

froschdesign commented on docs/book/v3/quick-start.md:
```diff
@@ -1,781 +1,435 @@
-laminas-view provides the "View" layer of Laminas's MVC system. It is a
-multi-tiered system allowing a variety of mechanisms for extension,
-substitution, and more.
+`laminas-view` provides a "View" layer for PHP applications.
```
Change the content on this page as necessary, and I would then divide it up into different pages later.

gsteel committed:
There is no longer a view event

Signed-off-by: George Steel <george@net-glue.co.uk>

gsteel committed:
WIP on template resolver docs

Signed-off-by: George Steel <george@net-glue.co.uk>

gsteel committed:
Re-work view script documentation

Signed-off-by: George Steel <george@net-glue.co.uk>

gsteel committed:
Update the view helpers intro page

Signed-off-by: George Steel <george@net-glue.co.uk>

froschdesign commented on docs/book/v3/helpers/intro.md:
```diff
@@ -1,53 +1,48 @@
+use Laminas\View\Helper\EscapeHtml;
+use Laminas\View\HelperPluginManager;
+
+/** @var ServiceManager $container */
```
`ServiceManager` or any PSR-11 container?

froschdesign replied:
Maybe we should add a full example with the `ServiceManager` in a details disclosure element: https://docs.laminas.dev/laminas-view/v2/application-integration/stand-alone/#create-layout-model-and-render-output

gsteel committed:
Improve helpers into docs to be less biased to laminas service manager

Signed-off-by: George Steel <george@net-glue.co.uk>

gsteel committed:
Re-work `PhpRenderer` docs

Signed-off-by: George Steel <george@net-glue.co.uk>

gsteel committed:
Drop 'standalone' application integration docs

Laminas view _is_ entirely standalone. Now the rest of the docs don't refer to MVC, this page is unnecessary.

Signed-off-by: George Steel <george@net-glue.co.uk>

gsteel commented:
I think this is ready for a final review @froschdesign 
Please feel free to push directly to the PR.

I still have work to do on the migration guide and the only other thing that might be lacking is a more in-depth guide to view models perhaps.

froschdesign reviewed (approved):
Good rework and cleanup! 👍🏻 
(I will add my changes in separate pull requests.)
